monkeyhex.py

```diff
@@ -2,14 +2,22 @@
 from future.utils import iteritems
 from past.builtins import long
 
+HEX_THRESHOLD = 10**17 # since that hex is more succinct ...
+alwaysHex = True
+
+def _hex(x):
```

Comment:
name this function more descriptively, possibly `conditional_hex`

Reply:
OK